Resources/Locale/en-US/silicons/derelict-cyborg-role.ftl

```diff
@@ -0,0 +1 @@
+derelict-cyborg-round-end-agent-name = derelict cyborg
```

Comment:
remove this file

Reply:
i thought i had removed that already. strange.

Reply:
the file does not appear on my system.

Reply:
I think i forgot to push that.